packages/svelte-materialify/@types/FluidGrid.d.ts

```diff
@@ -0,0 +1,40 @@
+import { SvelteComponent } from 'svelte-materialify/@types/shared';
+
+interface FluidGridProps {
+  /** whether this is a container */
+  container?: boolean;
+  /** whether this is an item */
+  item?: boolean;
+  /** amount of grid spacing */
+  spacing?: number;
+  /** sets direction of grid grid */
+  direction?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
+  /** sets justification of grid items */
+  justify?:
+    | 'start'
+    | 'center'
+    | 'end'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly';
+  /** sets alignment of grid items */
+  alignItems?: 'start' | 'center' | 'end' | 'stretch' | 'baseline';
+  /** sets column size at xs-breakpoint */
+  xs?: number | boolean;
```

Comment:
I'm unsure about the `number | boolean` types.
actually the `number` only accepts a certain range (1 through `$grid-columns`).
and the `boolean` only accepts `true`.
can that be expressed somehow? I'm new to typescript ;)